### ISSUE TITLE
Permalinks for variables and causal links

### ISSUE THREAD

**Issue by xrotwang** [CHIELDOnline/CHIELD#155]
Currently variables have unique URLs of the form https://chield.excd.org/variable.html?key=611107992
I'm not sure these are meant to be permalinks - but think they should be, e.g. to be able link from datasets to relevant variables. Ideally, I think, also individual causal links should have persistent URLs, again to be able create links (or RDF triples) of the type "paper x corroborates causal link y" or "dataset y contradicts causal link y".

**Comment by seannyD** [comment]
Good idea.  I'm not entirely sure how to do this, since at the moment the keys are just a hash of the variable name, and the variable name might change.  I'll look into permalinks.  The other thing I have to do is make a separate table for variable definitions, but need to consult some people about the best approach, theoretically.
